src/vstest.console/Internal/MSBuildLogger.cs

```diff
@@ -197,49 +222,157 @@ private static bool TryGetStackFrameLocation(string stackFrame, out string? line
             line = match.Groups["line"].Value;
         }
 
-        Debug.WriteLine($">>>> {(match.Success ? "MATCH" : "NOMATCH")} {stackFrame}");
-
         return match.Success;
     }
 
+    /// <summary>
+    /// Writes message to standard output, with the name of the message followed by the number of
+    /// parameters. With each parameter delimited by '||||', and newlines replaced with ~~~~ and !!!!.
+    /// Such as:
+    ///  ||||run-start1||||s:\t\mstest97\bin\Debug\net8.0\mstest97.dll
+    ///  ||||test-failed6||||TestMethod5||||Assert.IsTrue failed. ||||   at mstest97.UnitTest1.TestMethod5() in s:\t\mstest97\UnitTest1.cs:line 27~~~~!!!!   at Syste...
+    /// </summary>
+    /// <param name="name"></param>
+    /// <param name="data"></param>
+    private static void SendMessage(string name, params string?[] data)
+    {
+        TPDebug.Assert(Output != null, "Initialize should have been called");
+
+        var message = FormatMessage(name, data);
+        Debug.WriteLine($"MSBUILDLOGGER: {message}");
```

Comment:
Remove?

Reply:
This won't compile into prod, and I want to be using this locally for debugging possible issues we introduce. Imho good to go.